include/sentry.h

```diff
@@ -1362,6 +1362,14 @@ SENTRY_API sentry_user_consent_t sentry_user_consent_get(void);
  */
 SENTRY_API sentry_uuid_t sentry_capture_event(sentry_value_t event);
 
+struct sentry_path_s;
+typedef struct sentry_path_s sentry_path_t;
+
+/**
+ *  This function allows for independently created minidumps to be captured.
```

Comment:
This could be simplified, e.g.
```suggestion
 *  Allows capturing independently created minidumps.
```